include/mqt-core/dd/Package.hpp

```diff
@@ -2006,24 +1961,18 @@ template <class Config> class Package {
   ///
   mEdge reduceAncillae(mEdge& e, const std::vector<bool>& ancillary,
```

Comment:
Unfortunately, GitHub won't let me comment on the respective function, but the `reduceGarbage` method (for matrices) should also be adapted to properly handle the new identity-less DD structure.